coreos/master-config.yaml

```diff
@@ -22,13 +22,15 @@ write_files:
     permissions: 0755
     content: |
       #! /usr/bin/bash
+      # network not always up, ping to wait for internet connection
+      ping google.com -c 5
```

Comment:
Don't think this is the right way to do this.

Firstly, it isn't a guarantee the network it up - it could time out 5 times and then carry on.

If we want to do something like this, we probably want to continue until connection is verified.  However, a better solution might just be making sure this script is executed at the correct point in the startup process (or that it is re-tried when it fails)